src/nbl/video/COpenGL_Queue.h

```diff
@@ -245,6 +257,12 @@ class COpenGL_Queue final : public IGPUQueue
                     _state.ctxlocal.removePipelineEntry(&gl, hash);
                 }
                 break;
+                case ERT_BEGIN_CAPTURE:
+                    m_rdoc_api->StartFrameCapture(NULL, NULL);
+                break;
+                case ERT_END_CAPTURE:
+                    m_rdoc_api->EndFrameCapture(NULL, NULL);
+                break;
```

Comment:
@Crisspl this wont work
> RENDERDOC_DevicePointer and RENDERDOC_WindowHandle are described above in SetActiveWindow(). device and wndHandle can either or both be set to NULL to wildcard match against active device/window combinations. This wildcard matching can be used if the handle is difficult to obtain where frame captures are triggered.

> For example if device is NULL but wndHandle is set, RenderDoc will begin a capture on the first API it finds that is active on that window.

> If the wildcard match has multiple possible candidates, it is not defined which will be chosen. Wildcard matching should only be used when e.g. it is known that only one API is active on a window, or there is only one window active for a given API.

> If no window has been created and all rendering is off-screen, NULL can be specified for the window handle and the device object can be passed to select that API. If both are set to NULL, RenderDoc will simply choose one at random so is only recommended for the case where only one is present.


You should at least pass in `RENDERDOC_DevicePointer` which is
> For OpenGL it must be the HGLRC, GLXContext, or EGLContext context object.
> For OpenGLES it must be the EGLContext context object.

Reply:
fixed